lib/parser.js

```diff
@@ -38,16 +38,31 @@ function RobotsParser (url, options, after_parse) {
   this.statusCode = -1;
   this.redirectLimit = 5;
   this.allowAll = false;
-  if (null === options){
+/*   if (null === options){
     this.options = {};
   } else if (typeof options === 'string'){
     this.options = { headers: {userAgent:options}};
   } else if (typeof options === 'object'){
     this.options = options;
+  } */
```

Comment:
Skip commenting out this code, better for diff and everything to just remove it instead